tests/unit/test_service_steps.py

```diff
@@ -676,8 +676,8 @@ def test_emr_modify_instance_group_by_name_step_creation():
 
 
 @patch.object(boto3.session.Session, 'region_name', 'us-east-1')
-def test_databrew_start_job_run_step_creation_sync():
-    step = GlueDataBrewStartJobRunStep('Start Glue DataBrew Job Run - Sync', parameters={
+def test_databrew_start_job_run_step_creation_default():
+    step = GlueDataBrewStartJobRunStep('Start Glue DataBrew Job Run - Default', parameters={
```

Comment:
```suggestion
    step = GlueDataBrewStartJobRunStep('Start Glue DataBrew Job Run - default', parameters={
```

nit: address before pushing if you agree